src/include/miopen/invoke_params.hpp

```diff
@@ -100,6 +100,16 @@ struct AnyInvokeParams
         return *reinterpret_cast<const std::remove_cv_t<Actual>*>(impl->GetRawPtr());
     }
 
+    template <class Actual>
+    std::remove_cv_t<Actual>& CastTo() const
```

Comment:
This needs to be `const std::remove_cv_t<Actual>&` otherwise this will always drop the const